### ISSUE TITLE
Fix summing operation when MPI SIZE > N_y

### ISSUE THREAD

**Issue by espenhgn** [INM-6/hybridLFPy#20]
```
cell 17 population b5 in 4.33 s
cell 25 population b5 in 4.62 s
cell 1 population b5 in 4.65 s
Traceback (most recent call last):
  File "example_microcircuit.py", line 338, in <module>
    pop.collect_data()
  File "/p/project/cjinb33/jinb3323/sources/hybridLFPy/hybridLFPy/population.py", line 761, in collect_data
    lfp = self.calc_signal_sum(measure='LFP')
  File "/p/project/cjinb33/jinb3323/sources/hybridLFPy/hybridLFPy/population.py", line 623, in calc_signal_sum
    dtype=np.float32)
TypeError: 'float' object cannot be interpreted as an integer
```

**Comment by espenhgn** [comment]
Will fix with #17

**Comment by espenhgn** [comment]
Fixed by merging #9
